packages/okta-angular/test/e2e/harness/package.json

```diff
@@ -22,14 +22,14 @@
     "@angular/platform-browser": "^6.0.7",
     "@angular/platform-browser-dynamic": "^6.0.7",
     "@angular/router": "^6.0.7",
-    "@okta/okta-angular": "file:../../../okta-okta-angular-1.0.2.tgz",
+    "@okta/okta-angular": "file:../../../okta-okta-angular-v1.0.3.tgz",
```

Comment:
Should probably talk about this in a separate discussion, but am wondering if there's a better way to do this, i.e.
- There's a local test harness package
- There's a published library package that it tests
- Is the best way to set the dependency in the test harness to depend on a local reference to a hardcoded version tarball path?

Reply:
Yep. I was thinking about the same thing, where this PR lays the ground work to finding a much better way to run our tests.

With the shared workspace model, we can now have a shared test-suite that uses the same `page-objects` and specs. Right now, we're duplicating a significant amount of code.

Re: the `tgz` import:

This was a decision I made when originally publishing this library, where we would try to emulate a consumer app as close as possible. Over the last year, I've discovered that there are much better ways of doing this. 😄